### PR TITLE
[Related #33] Create input component and its stories

### PR DESCRIPTION
**Pull request by supunTE**
related #33 
- input component
- storybook story

### CODE REVIEW

File: apps/client/src/components/molecules/input/Input.component.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import cs from 'classnames';
+import React, { ReactElement, useState } from 'react';
+
+export interface InputProps {
+  value?: string;
+  placeholder?: string;
+  props?: React.InputHTMLAttributes<HTMLInputElement>;
+  isValid?: boolean;
+  isInvalid?: boolean;
```
Comment (dalanad) [inline apps/client/src/components/molecules/input/Input.component.tsx]: Why do we have a `isValid` and a `isInvalid`. What does `isValid` do ? shall we rename it give the proper meaning

**Review by dalanad — CHANGES_REQUESTED** [review]

File: apps/client/src/components/molecules/input/Input.component.tsx (hunk 1 above)
Comment (supunTE) [reply apps/client/src/components/molecules/input/Input.component.tsx]: In UI we have 3 states for input,

1. Initial/Default state (Mostly the empty state)
2. When the value is valid
3. When the value is invalid

image

**Review by dalanad — APPROVED** [review]

File: apps/client/src/components/molecules/input/Input.component.tsx (hunk 1 above)
Comment (pasinducw) [reply apps/client/src/components/molecules/input/Input.component.tsx]: If no 2 states can exist at the same time, shall we use an enum to indicate the state?
enum InputState {
   Initial,
   Valid,
   Invalid,
}

**Review by pasinducw — CHANGES_REQUESTED** [review]

File: apps/client/src/components/molecules/input/Input.component.tsx (hunk 1 above)
Comment (supunTE) [reply apps/client/src/components/molecules/input/Input.component.tsx]: Changed 14eb6d9e00d4f5a8eb5c43963a9013f2f86db5c2

**Review by pasinducw — APPROVED** [review]

**Pull request merged**